src/com/owncloud/android/utils/BitmapUtils.java

```diff
@@ -266,5 +274,41 @@ public static boolean isImage(File file) {
 
         return (mimeType != null && mimeType.startsWith("image/"));
     }
-    
+
+    /**
+     * calculates the RGB value based on a given account name.
+     *
+     * @param accountName The account name
+     * @return corresponding RGB color
+     * @throws UnsupportedEncodingException if the charset is not supported
+     * @throws NoSuchAlgorithmException if the specified algorithm is not available
+     */
+    public static int[] calculateRGB(String accountName) throws UnsupportedEncodingException, NoSuchAlgorithmException {
```

Comment:
more general name, sure. I made it public since I saw no harm in it just in case someone might need such a function in the future somewhere.